api/internal/adapters/grpc/service.go

```diff
@@ -47,36 +47,28 @@ func (s service) GetUserByPhone(ctx context.Context, phone string) (*core.ApiUse
 }
 
 func (s service) GetAccounts(ctx context.Context, ID string) ([]gomoney.Account, error) {
-	acc, err := s.acl.GetAccounts(ctx, &grpc3.StringID{Id: ID})
+	acc, err := s.acl.GetAccounts(ctx, &lgrpc.StringID{Id: ID})
 	if err != nil {
 		return nil, errors.Wrap(err, ErrServiceCall.Error())
 	}
 	accs := make([]gomoney.Account, len(acc.Accounts))
-	for i := 0; i < len(accs); i++ {
-		acci := acc.GetAccounts()[i]
-		accs[i] = gomoney.Account{
-			Id:          acci.GetId(),
-			Title:       acci.GetTitle(),
-			Description: acci.GetDescription(),
-			Balance:     acci.GetBalance(),
-			Currency:    gomoney.Currency(acci.GetCurrency().String()),
-			IsBlocked:   acci.GetIsBlocked(),
-		}
+	for i, acci := range acc.Accounts {
+		accs[i] = *mapPAccToAcc(acci)
 	}
 	return accs, nil
```

Comment:
This needs to be tested, because goland gave me linter warning when i tried to destructure a function that returns a pointer. Functions themselves are kinda objects